src/Features/Core/Portable/EditAndContinue/AbstractEditAndContinueAnalyzer.cs

```diff
@@ -5260,7 +5286,7 @@ private static bool TryMapParameter((SyntaxNode? Node, int Ordinal) parameterKey
             var newLambdaSymbol = GetLambdaExpressionSymbol(newModel, newLambda, cancellationToken);
 
             // signature validation:
-            if (!ParametersEquivalent(oldLambdaSymbol.Parameters, newLambdaSymbol.Parameters, exact: false))
+            if (!ParameterTypesEquivalent(oldLambdaSymbol.Parameters, newLambdaSymbol.Parameters, exact: false))
```

Comment:
I was confused by this method name. Don't we also consider a change to the parameter's ref kind a rude edit? Do we have existing tests for such scenarios?

Reply:
RefKind is part of type.